src/components/DecisioningEngine/createContextProvider.js

```diff
@@ -9,16 +9,77 @@ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTA
 OF ANY KIND, either express or implied. See the License for the specific language
 governing permissions and limitations under the License.
 */
-export default ({ eventRegistry }) => {
-  const globalContext = {}; // holder of global context like current time/date, browser name/version, day of week, scroll position, time on page, etc
+import getBrowser from "../../utils/getBrowser";
+import parseUrl from "../../utils/parseUrl";
+
+export default ({ eventRegistry }, currentWindow) => {
```

Comment:
I prefer window to be moved within the object, instead of a second parameter.  And it can just be called `window`

```js
export default ({ eventRegistry, window }) => {
```